tests/EpisodeIntegrationServiceTests/ReceiveDataTests/ReceiveDataTests.cs

```diff
@@ -100,7 +101,7 @@ public async Task ReceiveData_Should_Map_Episode_To_EpisodeDto()
             EarlyRecallDate = DateOnly.Parse("2018-03-14"),
             CallRecallStatusAuthorisedBy = "SCREENING_OFFICE",
             EndCode = "SC",
-            EndCodeLastUpdated = DateTime.Parse("2020-03-31 00:00:00+01"),
+            EndCodeLastUpdated = DateTime.Parse("2020-03-31 00:00:00+01").ToUniversalTime(),
```

Comment:
Suggest removing the `+01` if it's not needed:

```suggestion
            EndCodeLastUpdated = DateTime.Parse("2020-03-31 00:00:00").ToUniversalTime(),
```